.github/renovate.json5

```diff
@@ -60,6 +60,7 @@
         "minor",
       ],
       enabled: false,
+      automerge: true,
```

Comment:
do you want to automerge for both `main` and release branches?

Reply:
you are right, i need to enable automerge for all package rules, and yes i would also auto-merge release branches